src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/adjudications/AdjudicationsApiService.kt

```diff
@@ -21,7 +21,7 @@ class AdjudicationsApiService(private val adjudicationsApiWebClient: WebClient)
   suspend fun getAdjudicationsByBookingId(bookingId: Long, prisonIds: List<String>): List<ReportedAdjudicationDto> {
     // TODO - switch to simplified non-paged version when available
     return adjudicationsApiWebClient.get()
-      .uri("/reported-adjudications/booking/{bookingId}?agency={agency}&page={page}&size={size}", bookingId, prisonIds.joinToString(), 0, 1000)
+      .uri("/reported-adjudications/booking/{bookingId}?agency={agency}&status=ACCEPTED,REJECTED,AWAITING_REVIEW,RETURNED,UNSCHEDULED,SCHEDULED,REFER_POLICE,REFER_INAD,REFER_GOV,PROSECUTION,DISMISSED,NOT_PROCEED,ADJOURNED,CHARGE_PROVED,QUASHED,CORRUPTED&page={page}&size={size}", bookingId, prisonIds.joinToString(), 0, 1000)
       .retrieve()
```

Comment:
this API call will be replaced with a simpler one soon - for now make sure we get a addictions in case migrated ones sendup in a weird state